newIDE/app/src/UI/Theme/index.js

```diff
@@ -1,23 +1,61 @@
 // @flow
+import {
+  select,
+  blend as Install,
+} from '../../Utils/Object';
+import Language from '../../Utils/i18n/language';
+import { stash } from './domain';
```

Comment:
This stash is a global variable. Is there a way we could avoid:
- using a `Stash`. Can we replace it with a single variable?
- using a variable entirely? :) Could we just call `isLtr(lang)` whenever we need it?